### ISSUE TITLE
Error parsing_Cannot visualize genes

### ISSUE THREAD

**Issue by Puljkiae** [cogitokat/genegraphics#9]
Hello,

The Gene Graphics is a very useful tool and thank you for all the work. 

The new website was working normally (I don't remember when I last uploaded some TSV files but remember it was on the new website) and today I tried again and there is an error message -  Error parsing: File does not include required columns.
I tried uploading old TSV files (from SEED) which worked perfectly in the past, and they also failed to import. 

If I am doing something wrong let me know. Thank you very much!

**Comment by katlabs** [comment]
Hi,
Unfortunately, I am having trouble reproducing your issue with files I have. Can you paste a sample of a file that does not work for you?

**Comment by Puljkiae** [comment]
Dear Katherine,

 

During the weekend, I rearranged the data in the TSV file that were missing, such as genome and strand. So the issue is resolved. 

 

Also, I noticed if I put only + in the strand column when visualized it reads the correct directions of the gene, but if I put the + or – where it should be, it reads all like the plus strand. If only – is in put, then it all reads as reverse strand.

 

 

 

Ana Puljko, mag.ing.agr.

Laboratorij za okolišnu mikrobiologiju i biotehnologiju

Zavod za istraživanje mora i okoliša

Institut Ruđer Bošković

Bijenička cesta 54

HR-10000 Zagreb
